lib/bolt/project.rb

```diff
@@ -167,5 +167,131 @@ def check_deprecated_file
         Bolt::Logger.deprecation_warning('Using project.yaml instead of bolt-project.yaml', msg)
       end
     end
+
+    def migrate_project(configured_inventory_path, outputter)
+      migrating_inv = configured_inventory_path || inventory_file
+      inventory_1_to_2(migrating_inv, outputter) if migrating_inv.file?
+      bolt_yaml_to_bolt_project(migrating_inv, outputter)
+    end
+
+    def backup_file(origin_path, outputter)
```

Comment:
```suggestion
    private def backup_file(origin_path, outputter)
```

Reply:
I was thinking this might be useful elsewhere, but for now it's probalby better to leave a comment that this doesn't strictly need to be private